src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+
+    private static final FigureSupplier figureSupplies = new FigureSupplier();
+
+    public static void main(String[] args) {
+
+        Figure getRandomFIgure = figureSupplies.getRandomFigure();
+        Figure getDefaultFigure = figureSupplies.getDefaultFigure();
+
+        Figure[] array = {getRandomFIgure, getDefaultFigure};
```

Comment:
Creating an array to store figures for printing is unnecessary. You can print the figures directly within the loop where you generate them.